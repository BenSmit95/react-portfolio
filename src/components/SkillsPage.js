import React from 'react';

const SkillsPage = () => (
    <div>
        SkillsPage component
    </div>
);

export default SkillsPage