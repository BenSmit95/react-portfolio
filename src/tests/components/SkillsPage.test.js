import React from 'react';
import { shallow } from 'enzyme';
import SkillsPage from '../../components/SkillsPage';

test('Should render SkillsPage component correctly', () => {
    const wrapper = shallow(<SkillsPage />);
    expect(wrapper).toMatchSnapshot();
});