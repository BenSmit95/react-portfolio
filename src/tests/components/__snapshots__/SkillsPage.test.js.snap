// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render SkillsPage component correctly 1`] = `
<div>
  SkillsPage component
</div>
`;
